package/MDAnalysis/coordinates/RDKit.py

```diff
@@ -512,9 +512,25 @@ def _infer_bo_and_charges(mol):
     R-C(-O)-O the first oxygen read will receive a double bond and the other
     one will be charged. It will also affect more complex conjugated systems.
     """
+    atoms = sorted([a for a in mol.GetAtoms() if a.GetAtomicNum() > 1],
```

Comment:
Will this cause issues for dummy atoms etc? e.g. ``rdkit.Chem.MolFromSmiles(r"C(=C(/C([2*:2]))N)(\[1*:1])"))``

Reply:
I haven't considered dummy atoms when making the converter. It will very likely break the bond-order inferring since it needs the valence(s) of an atom to work properly but it's `-1` for dummies. Not sure if it would be useful or not